### ISSUE TITLE
Export Multiple Folders as MBOX Files

### ISSUE THREAD

**Issue by DrBrynzo** [thundernest/import-export-tools-ng#488]
Is there any way to export multiple folders from the top level to MBOX files? I used to just multi-select the folders in Thunderbird and export and I'd end up with an MBOX per folder in the destination. 

The lack of multi-select for folders (ctrl-click or shift-click) seems to be a new thing in the Thunderbird UI post v105, so that part is not a IET-NG problem.

I would still really like a way to export all my top-level folders (I do not use subfolders) to individual MBOX files without having to export one at a time. For example, a dialogue with check boxes for which folders to export to MBOX files or just "all folders".

Also, I realize that Thunderbird for now seems to be still using MBOX as the actual folder storage but just exporting the profile includes lots of extraneous files that can confuse other software just looking for MBOX files.

**Comment by cleidigh** [comment]
@DrBrynzo 
Yes I can't tell you how annoyed I was when the UI team decided multiple folder selection was gone.
As you have seen I did a complete redesign of the menus, I think for the better. I did, however, make some wrong assumptions that I am working on for v14.0.1

In v14.0.0 you can right click on the account level and do an Export Account 
That will export all account folders as mbox into a container directory. You should download the current beta as I already fixed several things with Export Account.

I am also going to add back several other functions at the account level for v14.0.1

Track and download beta here: 
- #466 

Let me know how you fair.
@cleidigh

**Comment by cleidigh** [comment]
@DrBrynzo 
This is enabled in b4
- #466 
@cleidigh

**Comment by DrBrynzo** [comment]
Great improvement, thanks. The only minor annoyance is that this exports everything, including Spam, Trash, and zero-size files for folders (i.e. [Gmail] in Gmail accounts). Having to manually delete those is a LOT better than having to select and export individual folders for sure.

**Comment by bellaceragit** [comment]
@cleidigh good morning. I agree with your disappointment and I also appreciate the improvement you've done.

I was used to do this kind of overall export / backup, as described above, by year of the e-mail. It may make sense or not: to me, for my needs it makes. Is it possible to do it already, or may you plan a further improvement in which one can export the e-mails from the top level filtering them for one specific year?

Thanks and best regards.
